### DISCUSSION TITLE
Mass Recipe Tag/Category Editor

### DISCUSSION THREAD

**Discussion by nubzzz** [mealie-recipes/mealie#4767] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

This would allow admins to quickly add tags and categories to existing recipes.

### Please provide a concise description of the feature that would resolve your issue.

I think that the best place for this would probably be under Household Statistics > (Tags|Categories). The hamburger menu for each could have a "Bulk Add" option that takes you to a page similar to the main recipes page with a checkbox for each recipe to add the tag or category to it.

### Please consider and list out some caveats or tradeoffs made in your design decision

I don't see any real caveats to this other that where it's placed within the UI.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This can be done from the data management page:
![image](https://github.com/user-attachments/assets/88b6e96e-a966-4b62-9c39-30b7f849f7ad)

https://demo.mealie.io/group/data/recipes

**Reply by nubzzz** [reply]
Thank you so much! This makes management much easier.

**Comment by nikomiko** [comment]
But there is no way to filter on this page. 
Let's say I have recipe with "cake" in the title : how can I batch assign a category like "desserts" ?
